### PR TITLE
firefox: prepare for sideloading in recent Firefox

### PR DESCRIPTION
**Pull request by rycee**

### CODE REVIEW

File: modules/misc/news.nix [hunk 1]
```diff
@@ -1381,6 +1381,32 @@ in
           A new module is available: 'services.lieer'.
         '';
       }
+
+      {
+        time = "2020-03-15T16:55:28+00:00";
+        condition = config.programs.firefox.enable;
+        message = ''
+          In preparation of Firefox's dropping support for extension
+          sideloading[1] we now install extensions directly in Firefox
```
Comment (cole-h) [inline modules/misc/news.nix]:
```suggestion
          In anticipation of Firefox dropping support for extension
          sideloading[1], we now install extensions directly to Firefox
```

(that `to` change is optional; looks/sounds better to me but no real difference either way)

File: modules/programs/firefox.nix [hunk 2]
```diff
@@ -23,8 +23,15 @@ let
     then "${firefoxConfigPath}/Profiles"
     else firefoxConfigPath;
 
+  # The extensions path shared by all profiles, will not be supported
```
Comment (cole-h) [inline modules/programs/firefox.nix]:
```suggestion
  # The extensions path shared by all profiles; will not be supported
```

Comment (rycee) [reply modules/programs/firefox.nix]: Thanks a lot for the suggestions :+1:

**Comment by rycee** [comment]
Rebased to master in cc386e4b3b3dbb6fb5d02e657afeacf218911d96.

**Pull request closed without merging**